### PR TITLE
feat(coinmarket): hide fees column in quote list

### LINKED ISSUE #11783: Hide "All fees included" info in Trezor Suite - Buy
### Issue
In buy we’re showing to user information “Fees Included all fees” for all partners. This is not true anymore as some partners now include for example blockchain fee and some don’t.

### Solution
Before we design better solution and communicate with all partners to ideally provide separately offer and fees, we want to remove this information. Specific fees are still communicated in partner’s checkout.

Snímek obrazovky 2024-03-27 v 17 41 34

### PR DESCRIPTION
**Pull request by FreeWall**
## Related Issue

Resolve #11783

## Screenshots:
image

### CODE REVIEW

File: packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuote.tsx [hunk 1]
```diff
@@ -252,15 +247,6 @@ export const BuyQuote = ({ className, quote, wantCrypto }: QuoteProps) => {
                         />
                     </Value>
                 </Column>
-                <Column>
-                    <Heading>
-                        <Translation id="TR_BUY_FEES" />{' '}
```
Comment (AdamSchinzel) [inline packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuote.tsx:257]: This translation string can also be removed from [messages.ts](https://github.com/trezor/trezor-suite/pull/11801/files#diff-988baecac61429698981f7b86b979d1627f33f484918f65402c4b345b5c57cba)

**Review by AdamSchinzel — CHANGES_REQUESTED** [review]

File: packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuote.tsx (hunk 1 above)
Comment (FreeWall) [reply packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuote.tsx:257]: This one will be probably used again in next iteration

Comment (AdamSchinzel) [reply packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuote.tsx:257]: But right now we won't use us it and nightly checks for unused translations will fail

Comment (FreeWall) [reply packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuote.tsx:257]: ok, removed https://github.com/trezor/trezor-suite/pull/11801/commits/74ee4077cec2dcc0a40478479d33226839f16ba3

Comment (AdamSchinzel) [reply packages/suite/src/views/wallet/coinmarket/buy/offers/Offers/List/BuyQuote.tsx:257]: Thanks, please squash it and we can merge it

**Review by tomasklim — APPROVED** [review]
LGTM

Speaking of Suite team, we do not go to trade tab very often. It would be nice if you add screenshots of previous state and a new state next time 🙏 Thank you

**Review by AdamSchinzel — APPROVED** [review]
lgtm

**Pull request merged**